2025-11-01T02:58:18.6532457Z Print service container logs: 05c493e0adf44cf7be100285b7ee6999_postgres16_9b9b28
2025-11-01T02:58:18.6537055Z ##[command]/usr/bin/docker logs --details 18cc59eb6609c0ed816a831bb947c4c498acd5c0d21773179071d8c22a173d79
2025-11-01T02:58:18.6693902Z  initdb: warning: enabling "trust" authentication for local connections
2025-11-01T02:58:18.6694740Z  The files belonging to this database system will be owned by user "postgres".
2025-11-01T02:58:18.6695795Z  initdb: hint: You can change this by editing pg_hba.conf or using the option -A, or --auth-local and --auth-host, the next time you run initdb.
2025-11-01T02:58:18.6696903Z  2025-11-01 02:53:10.921 UTC [1] LOG:  starting PostgreSQL 16.10 (Debian 16.10-1.pgdg13+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 14.2.0-19) 14.2.0, 64-bit
2025-11-01T02:58:18.6697531Z  2025-11-01 02:53:10.922 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-11-01T02:58:18.6697943Z  2025-11-01 02:53:10.922 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2025-11-01T02:58:18.6698431Z  2025-11-01 02:53:10.923 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-11-01T02:58:18.6698928Z  2025-11-01 02:53:10.926 UTC [62] LOG:  database system was shut down at 2025-11-01 02:53:10 UTC
2025-11-01T02:58:18.6699602Z  2025-11-01 02:53:10.930 UTC [1] LOG:  database system is ready to accept connections
2025-11-01T02:58:18.6699992Z  2025-11-01 02:53:45.656 UTC [91] FATAL:  database "test" does not exist
2025-11-01T02:58:18.6700358Z  2025-11-01 02:58:11.009 UTC [60] LOG:  checkpoint starting: time
2025-11-01T02:58:18.6700673Z  This user must also own the server process.
2025-11-01T02:58:18.6700917Z  
2025-11-01T02:58:18.6701170Z  The database cluster will be initialized with locale "en_US.utf8".
2025-11-01T02:58:18.6701570Z  The default database encoding has accordingly been set to "UTF8".
2025-11-01T02:58:18.6701954Z  The default text search configuration will be set to "english".
2025-11-01T02:58:18.6702246Z  
2025-11-01T02:58:18.6702417Z  Data page checksums are disabled.
2025-11-01T02:58:18.6702640Z  
2025-11-01T02:58:18.6702917Z  fixing permissions on existing directory /var/lib/postgresql/data ... ok
2025-11-01T02:58:18.6703275Z  creating subdirectories ... ok
2025-11-01T02:58:18.6703566Z  selecting dynamic shared memory implementation ... posix
2025-11-01T02:58:18.6703887Z  selecting default max_connections ... 100
2025-11-01T02:58:18.6704157Z  selecting default shared_buffers ... 128MB
2025-11-01T02:58:18.6704427Z  selecting default time zone ... Etc/UTC
2025-11-01T02:58:18.6704686Z  creating configuration files ... ok
2025-11-01T02:58:18.6704934Z  running bootstrap script ... ok
2025-11-01T02:58:18.6705225Z  performing post-bootstrap initialization ... ok
2025-11-01T02:58:18.6705512Z  syncing data to disk ... ok
2025-11-01T02:58:18.6705826Z  
2025-11-01T02:58:18.6706270Z  
2025-11-01T02:58:18.6706622Z  Success. You can now start the database server using:
2025-11-01T02:58:18.6707152Z  
2025-11-01T02:58:18.6707389Z      pg_ctl -D /var/lib/postgresql/data -l logfile start
2025-11-01T02:58:18.6707676Z  
2025-11-01T02:58:18.6708217Z  waiting for server to start....2025-11-01 02:53:10.684 UTC [48] LOG:  starting PostgreSQL 16.10 (Debian 16.10-1.pgdg13+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 14.2.0-19) 14.2.0, 64-bit
2025-11-01T02:58:18.6708958Z  2025-11-01 02:53:10.685 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-11-01T02:58:18.6709679Z  2025-11-01 02:53:10.688 UTC [51] LOG:  database system was shut down at 2025-11-01 02:53:10 UTC
2025-11-01T02:58:18.6710138Z  2025-11-01 02:53:10.692 UTC [48] LOG:  database system is ready to accept connections
2025-11-01T02:58:18.6710453Z   done
2025-11-01T02:58:18.6710620Z  server started
2025-11-01T02:58:18.6710796Z  
2025-11-01T02:58:18.6711086Z  /usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*
2025-11-01T02:58:18.6711443Z  
2025-11-01T02:58:18.6711663Z  2025-11-01 02:53:10.802 UTC [48] LOG:  received fast shutdown request
2025-11-01T02:58:18.6712368Z  waiting for server to shut down....2025-11-01 02:53:10.803 UTC [48] LOG:  aborting any active transactions
2025-11-01T02:58:18.6712959Z  2025-11-01 02:53:10.805 UTC [48] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
2025-11-01T02:58:18.6713421Z  2025-11-01 02:53:10.805 UTC [49] LOG:  shutting down
2025-11-01T02:58:18.6713756Z  2025-11-01 02:53:10.805 UTC [49] LOG:  checkpoint starting: shutdown immediate
2025-11-01T02:58:18.6714748Z  2025-11-01 02:53:10.808 UTC [49] LOG:  checkpoint complete: wrote 3 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.001 s, sync=0.001 s, total=0.003 s; sync files=2, longest=0.001 s, average=0.001 s; distance=0 kB, estimate=0 kB; lsn=0/14F2AD8, redo lsn=0/14F2AD8
2025-11-01T02:58:18.6715592Z  2025-11-01 02:53:10.812 UTC [48] LOG:  database system is shut down
2025-11-01T02:58:18.6715893Z   done
2025-11-01T02:58:18.6716048Z  server stopped
2025-11-01T02:58:18.6716221Z  
2025-11-01T02:58:18.6716431Z  PostgreSQL init process complete; ready for start up.
2025-11-01T02:58:18.6716709Z  
2025-11-01T02:58:18.6722631Z Stop and remove container: 05c493e0adf44cf7be100285b7ee6999_postgres16_9b9b28
2025-11-01T02:58:18.6727059Z ##[command]/usr/bin/docker rm --force 18cc59eb6609c0ed816a831bb947c4c498acd5c0d21773179071d8c22a173d79
2025-11-01T02:58:18.8140331Z 18cc59eb6609c0ed816a831bb947c4c498acd5c0d21773179071d8c22a173d79
2025-11-01T02:58:18.8168370Z Remove container network: github_network_c13f4be75e0b4307847490f96dd1634e
2025-11-01T02:58:18.8173299Z ##[command]/usr/bin/docker network rm github_network_c13f4be75e0b4307847490f96dd1634e
2025-11-01T02:58:18.9435435Z github_network_c13f4be75e0b4307847490f96dd1634e
